ts/src/renderer/phaser/classes/PhaserFloatingText.ts

```diff
@@ -0,0 +1,43 @@
+class PhaserFloatingText extends Phaser.GameObjects.Text {
+
+	constructor (
+		scene: Phaser.Scene,
+		data: {
+			text: string,
+			x: number,
+			y: number,
+			color: string
+		},
+		unit?: PhaserUnit
+	) {
+		super(scene, data.x, data.y, data.text, { fontFamily: 'Verdana' });
+
+		this.setOrigin(0.5);
+		this.setFontSize(16);
+		this.setFontStyle('bold');
+		this.setFill(data.color || '#fff');
```

Comment:
we make sure to always pass a color so we don't need this `||`

Unit.js:1225, ClientNetworkEvents.js:151

Reply:
what if this class needs to be used elsewhere without passing color? I see no issue with having a default value, especially if it is done like that in pixi

Reply:
Ok i understand your argument. Currently there is no other place in the code where we call it so I felt it was unnecessary. Also if we do pass a value but it corrupts somehow, both defaults are white so we won't know which one has the error

Reply:
Yeah ok sold

Reply:
I checked and `setFill()` takes parameter of type `any` so I don't think we would have the problems I was thinking about. I was recalling `fillStyle()` we used from `Graphics` in region that specifically required a type `number`